### DISCUSSION TITLE
Set "default" language for an item

### DISCUSSION THREAD

**Discussion by federicocappellotto97** [directus/directus#9827] [Feature Requests]
I have some translatable fields, and when i set the interface template like this: `{{title}}`, '--' is shown, because there is no "default" language for that item. Maybe it would be a good idea to put a dropdown somewhere with languages, with one language selected, so all translatable fields are shown in that language and when i select another language all fields are shown in that other language.

(sorry for my english)

**Comment by azrikahar** [comment]
#8203 should be aimed at implementing this exact use case 🙌

**Reply by federicocappellotto97** [reply]
Ohh very good, thanks!

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
